CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ CHANGELOG
 - [sdk/dotnet] Add collection initializers for smooth support of Union<T, U> as element type
   [#5938](https://github.com/pulumi/pulumi/pull/5938)
 
+- Fix a bug in the Go SDK that could result in dropped resource dependencies.
```

Comment:
Was this intentional?

Reply:
No... I must have done something odd when merging from master. Let me fix that up.